### PR TITLE
Sagas testing 2

### PR DESCRIPTION
**Pull request by clfhhc**
setting up redux store: https://trello.com/c/oQgNkjl6/14-setup-redux-store
api calls: https://trello.com/c/ce69Lnn5/17-api-calls
redux-saga for async functions: https://trello.com/c/6XiiopLh/15-setup-redux-saga-for-async-actions

### CODE REVIEW

File: client/src/js/apis/fetchData.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// client/src/js/apis/fetchData.js
+
+import fetch from 'cross-fetch';
+
+const roomUrl = 'http://localhost:3003/rooms/';
+
+const getRoomInfo = async (roomId, NumberReviewsPerPage) => {
```
Comment (tonaramos) [inline client/src/js/apis/fetchData.js]: Is there a reason why NumberReviewsPerPage starts with uppercase?

File: client/src/js/apis/fetchData.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  try {
+    const url = new URL(roomUrl + roomId);
+    const params = { pageonly: 0, start: 0, limit: NumberReviewsPerPage };
```
Comment (tonaramos) [inline client/src/js/apis/fetchData.js]: should pageonly be camel case style?

File: client/src/js/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// client/src/js/reducers/index.js
```
Comment (tonaramos) [inline client/src/js/reducers/index.js]: Not sure I understand what this reducers file does, what do you need it for?

File: client/src/js/apis/fetchData.js (hunk 1 above)
Comment (clfhhc) [reply client/src/js/apis/fetchData.js]: no, I messed up with the camelCase here

File: client/src/js/apis/fetchData.js (hunk 2 above)
Comment (clfhhc) [reply client/src/js/apis/fetchData.js]: I'm not sure if the restful api will be case sensitive or not, so I just put them all in lower case for now.

File: client/src/js/reducers/index.js (hunk 3 above)
Comment (clfhhc) [reply client/src/js/reducers/index.js]: reducer is the key of Redux, which tells the redux store how it generates a new state based on the action object dispatched.

**Pull request merged**